types/twine-sugarcube/tslint.json

```diff
@@ -0,0 +1,5 @@
+{ "extends": "dtslint/dt.json",
+	"rules": {
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/twine-sugarcube/tslint.json;
  Line: 2;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.